src/audio/comp_module/module.c

```diff
@@ -0,0 +1,329 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2019 NXP. All rights reserved.
+//
+// Author: Paul Olaru <paul.olaru@nxp.com>
+
+#include <config.h>
+
+#if CONFIG_COMP_MODULE
+
+#include <stdint.h>
+#include <stddef.h> // what is it used for?
```

Comment:
There may be a lot files included, because they were needed in the past and no1 removed them. If you have no need for the header just remove it. However remember that the fact that it compiles without something doesn't mean that it shouldn't be included explicitly. F.e. if you need stdint types you should include it instead of depending of something else including it.

Reply:
Addressed, coming up in next push.